### ISSUE TITLE
Inconsistent spelling of focused/focussed in fxa-settings

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#14636]
## Description

fxa-settings inconsistently uses "focused" and "focussed"; and the Oxford English Dictionary describes the spelling focussed as irregular.

Although possibly living firm in the "nit" camp since I don't think any of these usages are user facing, so it's more of a DX papercut.

```sh
git grep -Eio "focus+ed" packages/fxa-settings | sort | uniq

packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:Focussed
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:focussed
packages/fxa-settings/src/components/Settings/FormPassword/index.tsx:Focused
packages/fxa-settings/src/components/Settings/InputText/index.tsx:Focussed
packages/fxa-settings/src/components/Settings/InputText/index.tsx:focussed
packages/fxa-settings/src/pages/ResetPassword/index.tsx:Focused
```

Per https://en.wiktionary.org/wiki/focussed

> ### Usage notes
> - The spelling [focused](https://en.wiktionary.org/wiki/focused) is much more common in the US but also more common in the UK and Australia.
> - The Oxford English Dictionary describes the spelling focussed as irregular.
> - The spelling focussed is a widely seen UK spelling but focused, (focuses, focusing) is more common in prestige publications.

### Steps to reproduce

```sh
git grep -Ein "focussed" packages | cat

packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:23:  const [focussed, setFocussed] = useState<boolean>(false);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:28:    setFocussed(true);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:29:  }, [setFocussed]);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:32:    setFocussed(false);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:33:  }, [setFocussed]);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:77:                        focussed ? 'shadow-input-blue-focus' : ''
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:80:                        focussed ? 'shadow-input-grey-focus' : ''
packages/fxa-settings/src/components/Settings/InputText/index.tsx:58:  const [focussed, setFocussed] = useState<boolean>(false);
packages/fxa-settings/src/components/Settings/InputText/index.tsx:62:    setFocussed(true);
packages/fxa-settings/src/components/Settings/InputText/index.tsx:73:    setFocussed(false);
packages/fxa-settings/src/components/Settings/InputText/index.tsx:92:        focussed ? 'border-blue-400 shadow-input-blue-focus' : 'border-grey-200'
packages/fxa-settings/src/components/Settings/InputText/index.tsx:103:            hasContent || focussed
```

```sh
git grep -Ein "focused" packages | wc -l # 55
git grep -Ein "focussed" packages | wc -l # 12
```

### Expected result

Switch everything to "focused"

### Actual result

focused: 55 usages
focussed: 12 usages

### Environment

Ye old <kbd>grep</kbd>.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-6447)

**Comment by vpomerleau** [comment]
I'm making some edits to `InputText` for https://github.com/mozilla/fxa/pull/14599 and `focussed` was bugging me. I'll go ahead and at least fix it there.

**Comment by pdehaan** [comment]
I think we still have one instance in packages/fxa-settings/src/components/Settings/Checkbox/index.tsx, `setFocussed`. Will open a new issue.

TLDR: `const [focused, setFocussed] = ...` (focused and focussed)

```sh
git grep -in focussed packages | cat

packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:23:  const [focused, setFocussed] = useState<boolean>(false);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:28:    setFocussed(true);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:29:  }, [setFocussed]);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:32:    setFocussed(false);
packages/fxa-settings/src/components/Settings/Checkbox/index.tsx:33:  }, [setFocussed]);
```
